src/views/MissionPlanningView.vue

```diff
@@ -716,44 +784,46 @@ const addWaypoint = (
   type: WaypointType,
   altitudeReferenceType: AltitudeReferenceType
 ): void => {
-  if (planningMap.value === undefined) throw new Error('Map not yet defined')
+  if (!isDragging.value) {
```

Comment:
Instead of checking if it's *not* dragging and moving all the lines below, could we check if it's being dragged instead? I could be wrong but it seems like it would prevent changes on around 50 lines of code.